scripts/strip_newlines.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+import sys
+import os
+
+sys.path.insert(0, './coq-tools')
+from strip_newlines import strip_newlines
+
+def getListOfFiles(dirName):
+    listOfFile = os.listdir(dirName)
+    allFiles = list()
+    for entry in listOfFile:
+        fullPath = os.path.join(dirName, entry)
+        if os.path.isdir(fullPath):
+            allFiles = allFiles + getListOfFiles(fullPath)
+        else:
+            allFiles.append(fullPath)
+    return allFiles  
+
+
+def main():
+    dirName = './theories/VLSM'
```

Comment:
This hard-coded path makes the script not portable across repos, as `casper-cbc-proofs` has `CBC`, `CBC_VLSM` and `VLSM_index` in the `theories/` directory.

```suggestion
    dirName = './theories'
```